tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDateOrder.java

```diff
@@ -46,7 +47,7 @@ public void TestDateImportance() {
         try {
             // Build test file
 
-            XMLSource source = new SimpleXMLSource("xx");
+            XMLSource source = new SimpleXMLSource(LocaleNames.XX_TEST);
```

Comment:
```suggestion
            XMLSource source = new SimpleXMLSource("mul");
```

mul seems good for here

Reply:
Again, let's consider that (with `LocaleNames.MUL` not a string literal) for a follow-up -- such a change is somewhat out-of-scope for this ticket, which is about example dependencies...